### PR TITLE
FT.DEBUG INDICES_LIST allows to get all existing indices

### LINKED ISSUE #1119: Ability to efficiently query the available indices and aliases
Currently solution exists by scanning the keyspace which is inefficient.
By the introduction of schema-based search, this option will become invalid.
In addition there is now way to fetch the existing aliases and which physical indices it points to (can be split up into second issue)
related issue: #622

### PR DESCRIPTION
**Pull request by ashtul**
fix #1119 
undocumented until added to coordinator

### CODE REVIEW

File: src/cursor.c [hunk 1]
```diff
@@ -355,3 +355,15 @@ void CursorList_Destroy(CursorList *cl) {
   rm_free(cl->specs);
   pthread_mutex_destroy(&cl->lock);
 }
+
+void CursorList_GetIndices(CursorList *cl, char ***indicesList, size_t *indicesCount) {
```
Comment (MeirShpilraien) [inline src/cursor.c]: use arr.h and return array.

**Comment by ashtul** [comment]
similar to #1005

**Comment by ashtul** [comment]
similar to #1005

File: src/cursor.c (hunk 1 above)
Comment (ashtul) [reply src/cursor.c]: @MeirShpilraien 
Done

**Comment by MeirShpilraien** [comment]
@ashtul let's wait until we will have a final definition of how to expose this information

**Comment by ashtul** [comment]
replaced by #1337

**Pull request closed without merging**